src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerSession.cs

```diff
@@ -356,28 +364,24 @@ private Task StartAnalysis(IDependencyChainNode<PythonAnalyzerEntry> node, Async
             }
         }
 
-        private void LogCompleted(IPythonModule module, Stopwatch stopWatch, TimeSpan startTime) {
-            if (_log != null) {
```

Comment:
These checks are here to prevent evaluation of all of the arguments and the string building if the logging is null (Alex added them in a previous PR).

Reply:
Yes, needed to merge.

Reply:
I think these still need to be restored.

Reply:
Done